qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -323,8 +323,9 @@ def convert_acquire(self, instruction):
 
         cmd = commands.Acquire(duration, discriminator=discriminator, kernel=kernel)
         schedule = Schedule()
-        schedule |= commands.AcquireInstruction(cmd, qubit_channels, mem_slots,
-                                                register_slots) << t0
+
+        for qubit_channel, mem_slot, reg_slot in zip(qubit_channels, mem_slots, register_slots):
+            schedule |= commands.AcquireInstruction(cmd, qubit_channel, mem_slot, reg_slot) << t0
```

Comment:
👍 Good! Maybe `acquire_channel` or simply `channel` instead of `qubit_channel`? We don't use it elsewhere